packages/design-system/src/styles/components/_Button.scss

```diff
@@ -259,31 +247,22 @@ $button-color-variants: (
     &.ds-c-button--hover {
       background-color: nth($color, 2);
     }
-
-    @if $ds-include-focus-styles {
-      &:focus,
-      &.ds-c-button--focus {
-        @include focus-styles;
-      }
-    } @else {
-      &:focus,
-      &.ds-c-button--focus {
-        background-color: nth($color, 2);
-      }
+    &:focus,
+    &.ds-c-button--focus {
+      @include focus-styles;
+      background-color: nth($color, 1);
```

Comment:
Needed to set the background color for focus to the same background color used for default states

Reply:
Can you talk about this more? I'm having trouble understanding what state this applies to

Reply:
This applies to an edge case where an a tag is used with button classes to be styles like a button. We want the background color for focus to be the same color as in the default state. 

For links the default focus background color on hover is white. We don't want that behavior with buttons.

Reply:
Primary button `<a>` tag with no background set 
![Screen Shot 2021-12-20 at 5 14 08 PM](https://user-images.githubusercontent.com/1449852/146839733-bf05ead5-ae75-4d40-8f4a-b6a94c1f286a.png)

Primary button `<a>` tag with background set 
![Screen Shot 2021-12-20 at 5 15 02 PM](https://user-images.githubusercontent.com/1449852/146839806-2153b435-dd21-4549-b472-bccb79a68e69.png)

Reply:
Thank you, that makes sense to me 👍